pkg/apis/pipeline/v1beta1/workspace_validation.go

```diff
@@ -66,6 +66,11 @@ func (b *WorkspaceBinding) Validate(ctx context.Context) *apis.FieldError {
 		return apis.ErrMissingField("secret.secretName")
 	}
 
+	// For a CSI to work, you must provide the driver to use.
```

Comment:
We'll probably want to put this behind an [alpha feature flag](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#alpha-features) (and make this clear in the docs)